packages/salesforcedx-vscode-lwc/src/metasupport/metaSupport.ts

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2020, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as fs from 'fs';
+import * as vscode from 'vscode';
+import path = require('path');
+import semver = require('semver');
+
+const EXTENSION_NAME = 'salesforce.salesforcedx-vscode-lwc';
+
+export class MetaSupport {
+  private static instance: MetaSupport;
+  private static sfdxUri = '.sfdx';
+  private static resourceUri = path.join(MetaSupport.sfdxUri, 'resources');
+  private static lwcResourceUri = path.join(
+    MetaSupport.resourceUri,
+    'lwcResources'
+  );
+  private static dir = path.join(
+    vscode.workspace.rootPath!,
```

Comment:
`vscode.workspace.rootPath` is a deprecated API, you should update to use `vscode.workspace.workspaceFolders`

Reply:
unused variable removed.